Headlines/src/ScreenSwitcher/UIViewController+Switcher.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  UIViewController+Switcher.swift
+//  Canillitapp
+//
+//  Created by Ezequiel Becerra on 11/8/17.
+//  Copyright © 2017 Ezequiel Becerra. All rights reserved.
+//
+
+import UIKit
+
+extension UIViewController {
+    var controllerSwitcher: ScreenSwitcherViewController? {
+        get {
+            var parentViewController = self.parent
+            while parentViewController != nil && !parentViewController!.isMember(of: ScreenSwitcherViewController.self) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/6f1c7836a2c0d74d84b1a9f67b523ee54a5b5f94/Headlines/src/ScreenSwitcher/UIViewController+Switcher.swift#L15"></span>Line should be 120 characters or less: currently 121 characters</td>
    </tr>
  </tbody>
</table>